webpack/react_app/components/TargetingHosts/index.js

```diff
@@ -30,14 +30,14 @@ const WrappedTargetingHosts = () => {
   const [searchQuery, setSearchQuery] = useState('');
   const [pagination, setPagination] = useState({
     page: 1,
-    perPage,
+    per_page: perPage,
```

Comment:
the `perPage` and `perPageOptions` that was provided from core settings are already consumed in the core implementation, so you don't need to pass them twice.

Reply:
You're right, but to keep changes to a minimum I believe in "it works" approach for this PR :)